source/2020-06-05-the-ember-times-issue-151.md

```diff
@@ -46,14 +47,14 @@ Try out [`ember-await`](https://exelord.gitbook.io/ember-await/) today!
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Beta release of ember-data-model-fragments 🚢](https://github.com/lytics/ember-data-model-fragments/releases/tag/5.0.0-beta.0)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+ New beta alert! 🚨 Version 5.0.0-beta.0 of `ember-data-model-fragments` has recently been released!  
+ 
+ This beta provides support for Ember Data 3.13 + and includes a pretty serious rewrite to accommodate RecordData. Big shout out to the team behind it, and [Igor Terzic
```

Comment:
I think you wanted `+` to indicate `v3.13` or higher, and not Ember Data plus something else? I like to put the plus sign right after the version number, but feel free not to. :)

```suggestion
This beta provides support for Ember Data 3.13+ and includes a pretty serious rewrite to accommodate `RecordData`. Big shout out to the team behind it, and [Igor Terzic
```